setup/README.md

```diff
@@ -0,0 +1,74 @@
+# SteamPulse
+
+An integrated dashboard displaying trend analysis for new releases on steam
+
+## Configure environment
+
+```sh
+python3 -m venv venv
+source ./venv/bin/activate
+pip3 install -r requirements.txt
+```
+
+## Configure environment variables
+
+The following environment variables must be supplied in a `.env` file in the root directory, and a `terraform.tfvars` file in the setup directory:
+
+```sh
+ACCESS_KEY_ID     = XXXXXXXXXXX
+SECRET_ACCESS_KEY = XXXXXXXXXXX
+DATABASE_NAME     = steampulse
+DATABASE_USERNAME = steampulse_admin
+DATABASE_PASSWORD = XXXXXXXXXXX
+DATABASE_ENDPOINT = XXXXXXXXXXX
+```
+
+If hosting this service on AWS, you'll need to create your RDS to get the database endpoint.
+
+## Setup cloud resources
+
+**Warning** - AWS can incur unexpected costs, be sure you know what you're doing before replicating this section.
```

Comment:
I'm sure they will appreciate this warning! "We cannot be held liable for any costs incurred"